src/backend/input.rs

```diff
@@ -216,6 +231,9 @@ pub trait InputBackend: Sized {
     /// Get current `InputConfig`
     fn input_config(&mut self) -> &mut Self::InputConfig;
 
+    /// Called to inform the Input backend about a new focused Output for a `Seat`
+    fn set_output_metadata(&mut self, seat: &Seat, output: &Output);
```

Comment:
Ah, yes I see...
In this case, I'm thinking the output metadata should probably be a per-device property rather than per-seat. See, in a context of multi-output:

- pointer devices can change of "currently focused output", and as such the resolution to consider can change depending on where the pointer is in compositor space
- touchscreens on the other hand, are likely fixed to a single output, and their output resolution will not depend on where the pointer currently is
- tablet devices can, on x11, be configured to be either fixed with a single output, or cover the entire screen space covering all outputs. I'd like to keep this behavior.